src/main/java/oshi/software/os/mac/local/GlobalMemory.java

```diff
@@ -8,41 +8,68 @@
 package oshi.software.os.mac.local;
 
 import oshi.hardware.Memory;
-import oshi.util.ExecutingCommand;
+import oshi.software.os.mac.local.SystemB.VM_STATISTICS;
+import oshi.software.os.mac.local.SystemB.VM_STATISTICS64;
+
+import com.sun.jna.Platform;
+import com.sun.jna.ptr.IntByReference;
+import com.sun.jna.ptr.LongByReference;
 
 /**
- * Memory obtained by GlobalMemoryStatusEx.
+ * Memory obtained by host_statistics (vm_stat) and sysctl
  * 
- * @author alessandro[at]perucchi[dot]org
+ * @author widdis[at]gmail[dot]com
  */
 public class GlobalMemory implements Memory {
 
 	private long totalMemory = 0;

```

Comment:
The class variable and conditional in getTotal were already there, so I figured that was the intent.  I'll simplify.